README.md

```diff
@@ -71,12 +71,12 @@ import txt from 'raw-loader!./file.txt';
   <tbody>
     <tr>
       <td align="center">
-        <img width="150 height="150" src="https://github.com/sokra.png?s=150">
-        <br>
+        <img width="150" height="150" src="https://avatars.githubusercontent.com/sokra?v=3">

```

Comment:
To be consistent with the others project's README, also, https://github.com/sokra.png is a redirection to https://avatars.githubusercontent.com/sokra?v=3